site/AntDesign.Docs/Demos/Components/Table/demo/SaveLoadConfiguration.razor

```diff
@@ -0,0 +1,216 @@
+            new(32, "Miss Brown","New York No. 1 Lake Park",new DateTime(1989,1,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Reading),
+            new(33, "Sara Green","London No. 1 Lake Park",new DateTime(1989,12,1),true,Gender.Female,Hobbies.Reading ),
+            new(34, "Joe Black", "Sidney No. 1 Lake Park",new DateTime(1979,11,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Sport),
+            new(35, "White Red","London No. 2 Lake Park",new DateTime(1989,1,1),false,Gender.Male,null),
+            new(36, "John Anna","New York No. 1 Lake Park",new DateTime(1980,1,1),true,Gender.Male,Hobbies.Cooking),
+            new(37, "Anna Sunny","London No. 1 Lake Park",new DateTime(1983,12,1),true,Gender.Female,Hobbies.Reading | Hobbies.Sport),
+            new(38, "Joe Black", "Sidney No. 1 Lake Park",new DateTime(1977,11,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Sport),
+            new(39, "Jim Red","London No. 2 Lake Park",new DateTime(1985,1,1),false,Gender.Male,null),
+            new(40, "John Brown","New York No. 1 Lake Park",new DateTime(1980,1,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Reading),
+            new(41, "Sara Green","London No. 1 Lake Park",new DateTime(1983,12,1),true,Gender.Female,Hobbies.Reading | Hobbies.Sport),
+            new(42, "Anna White", "Sidney No. 1 Lake Park",new DateTime(1977,11,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Sport),
+            new(43, "Jim Red","London No. 2 Lake Park",new DateTime(1985,1,1),false,Gender.Male,null),
+            new(44, "John Miss","New York No. 1 Lake Park",new DateTime(1980,1,1),true,Gender.Male, Hobbies.Reading),
+            new(45, "Miss Green","London No. 1 Lake Park",new DateTime(1983,12,1),true,Gender.Female,Hobbies.Reading | Hobbies.Sport),
+            new(46, "Joe Black", "Sidney No. 1 Lake Park",new DateTime(1977,11,1),true,Gender.Male,Hobbies.Cooking ),
+            new(47, "Jim Sunny","London No. 2 Lake Park",new DateTime(1985,1,1),false,Gender.Male,null),
+            new(48, "White Anna","New York No. 1 Lake Park",new DateTime(1980,1,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Reading),
+            new(49, "Sara Green","London No. 1 Lake Park",new DateTime(1983,12,1),true,Gender.Female, Hobbies.Sport),
+            new(50, "Joe Anna", "Sidney No. 1 Lake Park",new DateTime(1977,11,1),true,Gender.Male,Hobbies.Cooking | Hobbies.Sport),
+            new(51, "Miss Red","London No. 2 Lake Park",new DateTime(1985,1,1),false,Gender.Male,null),
+        };
+
+    Table<Data> Table;
+    QueryModel SavedQueryModel;
+    string SavedQueryModelJson;
+
+    void OnChange(QueryModel<Data> query)
+    {
+        Console.WriteLine(JsonSerializer.Serialize(query));
+    }
+
+    void ResetTable()
+    {
+        Table.ResetData();
+    }
+
+    void SaveTable()
+    {
+        SavedQueryModel = Table.GetQueryModel();
+    }
+
+    void LoadTable()
+    {
+        Table.ReloadData(SavedQueryModel);
+    }
```

Comment:
I found that Filter does not restore state if you just set an object that has not had a serial number.

Reply:
Thank you for noticing the bug, I fixed it. Needed `[JsonConstructor]` annotation for JSON deserialization  is only available in net5, but is not available in netcoreapp3.1, so maybe for JSON deserialization should order of TargetFrameworks be `net5;netcoreapp3.1` in AntDesign.Docs.Server project; or to find some other solution. I am open to any advices or suggestions.

Reply:
@AnaNikolasevic: Shouldn't the JSON serialization of internal objects be handled outside of antdesign? So, for instance, remove the attribute and add a specific JsonConverter to your own project.

I mean, of course, it is convenient to directly serialize these objects, but this should not impact the dependency framework.